modal/README.md

```diff
@@ -0,0 +1,59 @@
+# Modal
+
+## Overview
+
+[Modal][1] lets you run generative AI models, large-scale batch jobs, job
+queues, and much more. The Datadog integration collects logs and metrics from
+your applications for you to observe directly in your Datadog account.
```

Comment:
Can you add a bit more context here about what data is  being sent in/what it represents, and why it's valuable for customers?